docs/tdvt.md

```diff
@@ -39,71 +55,52 @@ The TDVT tests for these things:
+2. Clone the [Connector Plugin SDK](https://github.com/tableau/connector-plugin-sdk/) and install TDVT.
+You can create an archive package and install that, or install from the live directory if you want to modify TDVT.
+3. Install TDVT
+   - We suggest you create and activate a venv:
 
         ```
         $ python -m venv ./tdvt-venv
         $ source tdvt-venv/bin/activate
         ```
 
-        or on Windows:
+   or on Windows:
 
         ```
         D:\src> python -m venv ./tdvt-venv
         D:\src> tdvt-venv/Scripts/activate.bat
         ```
-    You will know your venv is activated if (tdvt-venv) appears before your prompt. Alternatively, do a $which python to make sure it's pointing to a python executable in /tdvt-venv
+    You will know your venv is activated if (tdvt-venv) appears before your prompt. Alternatively, do a `$which python` to make sure it's pointing to a python executable in /tdvt-venv
 
-    Install TDVT:
+    Install TDVT in your virtual environment:
         ```
         $ (tdvt-venv) cd connector-plugin-sdk/tdvt
         $ (tdvt-venv) python -m pip install -e .
         ```
-        The . at the end is important. You can verify it installed by doing $ pip list
-
-Now run the following commands from the top level "tdvt" directory: 
-    - Create an archive package in the dist folder:
-`python setup.py sdist --formats gztar`
-    - Change directory to dist and install from the archived file:
-`python -m pip install tdvt-1.1.59.zip`
-   __Note:__ Instead of A and B, you can install the live version:
-    `python -m pip install -e .`
-    - Verify it is installed:
- `python -m pip list`
-
-1. Extract and then load the [TestV1 dataset](https://github.com/tableau/connector-plugin-sdk/tree/master/tests/datasets/TestV1) into your database.
+    You can then verify TDVT installed by running `pip list`:
+        ```
+         $ pip list
+
+         Package    Version Editable project location
+         ---------- ------- --------------------------------------------------------
+         defusedxml 0.7.1
+         pip        23.2.1
+         setuptools 65.5.1
+         tdvt       2.7.1   /path/to/connector-plugin-sdk/tdvt/tdvt
+         wheel      0.38.4
+        ```
+
+4. Extract and then load the [TestV1 dataset](https://github.com/tableau/connector-plugin-sdk/tree/master/tests/datasets/TestV1) into your database.
 __Notes:__
-    - See [Postgres Example](https://github.com/tableau/connector-plugin-sdk/tree/master/tests/datasets/TestV1/postgres/README.md) for an example of loading data into a database
+    - See the [Postgres Example](https://github.com/tableau/connector-plugin-sdk/tree/master/tests/datasets/TestV1/postgres/README.md) for an example of loading data into a database
     - See the following section about troubleshooting Boolean values if your database doesn’t have a native Boolean type.
```

Comment:
Either link to the FAQ below or call it out by name so they can find the note about boolean support.